sydent/db/threepid_associations.py

```diff
@@ -60,6 +64,34 @@ def getAssociationsAfterId(self, afterId, limit):
 
         return (assocs, maxId)
 
+    def removeAssociation(self, threepid, mxid):
+        cur = self.sydent.db.cursor()
+
+        cur.execute(
```

Comment:
why do we bother with this, ooi?

Reply:
Because it's a REPLACE INTO, so we don't need to insert an empty record if there's nothing there to start with